jerry-core/ecma/operations/ecma-objects-general.c

```diff
@@ -313,7 +313,7 @@ ecma_op_general_object_define_own_property (ecma_object_t *object_p, /**< the ob
   JERRY_ASSERT (property_desc_p->is_writable_defined || !property_desc_p->is_writable);
 
   /* 1. */
-  ecma_extended_property_ref_t ext_property_ref = {0,0};
+  ecma_extended_property_ref_t ext_property_ref = {{0},0};
```

Comment:
...and now Travis says this is wrong:

./jerry-core/ecma/operations/ecma-objects-general.c:316: error: brace should be placed on a separate line

...I assume that this is a style checker, objecting to the braces?  This code, however, looks like the right way to write it: am I not following a standard, or is there a bug in the style-checking code?